prisma/schema.prisma

```diff
@@ -0,0 +1,24 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model Story {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  content   String
+  postal    String
```

Comment:
Do we want to limit this to 3 characters on the DB layer?

```suggestion
  postal    String @db.VarChar(3)
```

Reply:
Oh dope - I hadn't dug into prisma validations. Is this documented?